src/CoreGraphics/CGColor.cs

```diff
@@ -309,6 +319,17 @@ static public CGColor CreateGenericGrayGamma2_2 (nfloat gray, nfloat alpha)
 			var h = CGColorCreateGenericGrayGamma2_2 (gray, alpha);
 			return h == IntPtr.Zero ? null : new CGColor (h);
 		}
+
+		[iOS (14,0)][TV (14,0)][Watch (7,0)][Mac (10,16)]
+		[DllImport(Constants.CoreGraphicsLibrary)]
+		static extern /* CGColorRef */ IntPtr CGColorCreateGenericCMYK (nfloat cyan, nfloat magenta, nfloat yellow, nfloat black, nfloat alpha);
+
+		[iOS (14,0)][TV (14,0)][Watch (7,0)][Mac (10,16)]
+		static public CGColor CreateCmyk (nfloat cyan, nfloat magenta, nfloat yellow, nfloat black, nfloat alpha)
+		{
+			var h = CGColorCreateGenericCMYK (cyan, magenta, yellow, black, alpha);
+			return h == IntPtr.Zero ? null : new CGColor (h);
```

Comment:
Shouldn't we use `Runtime.GetINativeObject<T>` here? I know we have `new CGColor` on other similar ones, I would not die on it but  `¯\_(ツ)_/¯`

Reply:
we could but it would not be optimal because `NSObject` is a `INativeObject` so the logic to create an instance from `GetINativeObject<T>` is much more costly (and there's no doubt here about which type should be used).